### ISSUE TITLE
Incorrect bone conversion in get_bone() functions

### ISSUE THREAD

**Issue by JekSun97** [Daylily-Zeleen/Godot-DragonBones#25]
I'm trying to draw a point at the position of the bone during animation, but it doesn't work for me, I tried using both the position of the bone itself and the animation bone position (get_bone_animation_position).

if the bone moves to the right during the animation, then my point does not move to the right, but goes down along the Y coordinate.

Try to display something in the position of the bone, it is unlikely to work, otherwise please share the correct code, since perhaps this is not an error at all.

**Comment by Daylily-Zeleen** [comment]
1. The `global_transform` of `DragonBonesBone` is the local transform of `DragonBoneArmature`.
2. The `transform` of `DragonBonesBone` is the local transform of parent bone.

`global_position`/`position`, `global_rotation`/`rotation` and `global_scale`/`scale` are similar to `global_transform`/`transform`.
These are same as bone's transform in `Skeleton3D` in godot.

To get the "real" global transform, you should use `global_transform` of `DragonBoneArmature` to do a transform:
```
    var bone : DragonBonesBone = ...
    var armature: DragonBoneArmature = ...
    var real_global_transform = armature.global_transform * bone.global_transform
```

**Comment by Daylily-Zeleen** [comment]
Fix by [a760ac2](https://github.com/Daylily-Zeleen/Godot-DragonBones/commit/a760ac2f872009392a1f8fd1a595e02491907c8a).

After this commit:
1. The `transform` of `DragonBonesBone` is a local transform of parent bone.
2. The `global_transform` of  `DragonBonesBone` is a local transform of its armature.

The `position`/`global_position`, `rotation`/`global_rotation` and `scale`/`global_scale` of  `DragonBonesBone` are same as `transform`/`global_transform` of `DragonBonesBone`.
This is same as bone infomation in `Skeleton3D` in godot.

If you want to get the "real" global pose of bone, you should use its armature's `global_transform` to transform:
```
    var bone :DragonBonesBone = ...
    var armature :DragonBonesArmature = ...
    var real_global_transform := armature.global_transform * bone.global_transform
```

**Comment by JekSun97** [comment]
For me it worked `armature.global_transform * bone.global_transform`
thanks!
